src/base_model.py

```diff
@@ -18,11 +20,25 @@ class BaseModel(Model):
     class Meta:
         """Set Database"""
 
-        database = db
+        database: SqliteDatabase = db
 
     class Schema(marsh.Schema):
         """The marshmallow schema all model schemas have to inherit from."""
 
+        id = marsh.fields.UUID()
+
+        @abc.abstractmethod
+        def _make(self, data: dict) -> "BaseModel":
+            """Constructs a model object from a dictionary."""
+            raise NotImplementedError()
```

Comment:
Should this method load the UUID? I think, that if that is not the case, the UUIDs will not be stable when reloading the objects.

Reply:
Or is that already done by a subclass method calling `return ModelTest(**data)`?
If so, could a method like that already be defined in the BaseModel?

Reply:
We shouldn't be able to define the id when initializing an object from a dict because we could overwrite existing rows in the database. When you want to update an object, run a query and update it 😅